### PR TITLE
Add gameplay menu

### PR DESCRIPTION
**Pull request by DavidPgl**
Very simple menu that allows to view settings in-game and leave tutorial and games

### CODE REVIEW

File: recursio-client/UI/Menus/StartMenu.gd [hunk 1]
```diff
@@ -80,7 +95,18 @@ func _ready():
 	get_tree().get_root().call_deferred("add_child", new_scene)
 
 
+func _process(_delta):
+	if not _in_game:
+		return
+	
+	if not Input.is_action_just_pressed("ui_cancel"):
+		return
```
Comment (kb173) [inline recursio-client/UI/Menus/StartMenu.gd:103]: This seems like a somewhat convoluted way to do it :sweat_smile: why not just react to `ui_cancel` for making it invisible?

File: recursio-client/UI/Menus/StartMenu.gd [hunk 2]
```diff
@@ -246,11 +272,29 @@ func _on_game_result_received(_winning_player_id):
...
 	_btn_play_tutorial.grab_focus()
+
+
+func _on_gameplay_menu_leave_pressed() -> void:
+	if not _in_game:
+		return
+	
+	if _world != null:
+		_return_to_game_room_lobby()
+	else:
```
Comment (kb173) [inline recursio-client/UI/Menus/StartMenu.gd:296]: This feels unsafe (are we really definitely in a tutorial if the world is null and we're allegedly in-game?), might want to at least add an assert

**Review by AmonShokhinAhmed — CHANGES_REQUESTED** [review]
Works fine I think, but maybe make the menu buttons a big bigger. 
Also opening it with the right face button seems weird. Should be on Start or Select or something. Additionally I cannot use the menu with the controller after opening it in the tutorial.

File: recursio-client/UI/Menus/StartMenu.gd (hunk 2 above)
Comment (DavidPgl) [reply recursio-client/UI/Menus/StartMenu.gd:296]: As `in_game` is only set when the world instantiated/freed (and same for the tutorial), it should be fine but will add them

File: recursio-client/UI/Menus/StartMenu.gd (hunk 1 above)
Comment (DavidPgl) [reply recursio-client/UI/Menus/StartMenu.gd:103]: Done

**Comment by DavidPgl** [comment]
Buttons are bigger but do not match with all the other buttons

**Comment by AmonShokhinAhmed** [comment]
My previous issues are fixed now, but opening the settings during the game, and then switching tabs breaks the focus (this does not happen on the titel screen).

**Comment by DavidPgl** [comment]
Fixed focus problems and some other smaller stuff

**Pull request merged**